assets/js/Line.jsx

```diff
@@ -95,6 +97,26 @@ function Line({
           readOnly={readOnly}
         />
       )}
+      {line === 'Silver' && (
+        <div>
+          <label htmlFor="chelsea_boarding">
```

Comment:
Since the input is nested inside the label, I don't think we need a `for`/`id` linkage here.

Reply:
yeah, i'd like to leave the `id` (mainly for testing), and it seems like no harm to leave the `htmlFor` here in that case.

Reply:
For the testing, rather than `find("input#chelsea_boarding")` does something like `find("input[name=chelsea_boarding]")` work? I think it's fine if we leave the ID, of course, this is more curiosity 🙂

Reply:
i'm sure it does.